packages/providers/src/provider.ts

```diff
@@ -227,13 +248,6 @@ export default class Provider {
     return getOperationsSdk(gqlClient);
   }
 
-  /**
-   * Connect provider to a different Fuel node url
-   */
-  connect(url: string) {
-    this.operations = this.createOperations(url);
-  }
-
```

Comment:
The new `connect` method is not to be confused with this _old_ method with the same name. I have preserved this functionality and put it under a different name - `updateUrl`.

Reply:
Could be possible that by updating the provider url we would have a provider with a different chain config? If so, maybe it makes sense to make `updateUrl` also async and to work similarly with `connect`, to fetch the `consesusParameters` once again, since they could be different from the last provider instance.